src/AVFoundation/AVAudioRecorder.cs

```diff
@@ -33,6 +33,12 @@
 
 namespace AVFoundation {
 #if !TVOS
+#if NET
+	[SupportedOSPlatform ("ios")]
+	[SupportedOSPlatform ("maccatalyst")]
+	[SupportedOSPlatform ("macos")]
+	[UnsupportedOSPlatform ("tvos")]
+#endif
 	public partial class AVAudioRecorder {
```

Comment:
`AVAudioRecorder`is a generated (partial) class, did your patch rewriting script miss this?

Reply:
Fixed.